templates/kapeta/block-type-cli/.env

```diff
@@ -0,0 +1,4 @@
+//#FILENAME:.env:create-only
```

Comment:
Should we use the pattern w/ `.env.sample` here as well maybe?

Reply:
With good docs/comments

Reply:
Uhm actually when is this used when there's the other one specifically for dev?

Reply:
`.env` is used when you run the built version of the CLI
`.env.development` is used when you run locally while developing. The `.env.development` is updated automatically by the `prepare:hosts` script when you run locally so it is not a "sample"... 🤔